pkg/server/grpc/tracee.go

```diff
@@ -345,17 +345,20 @@ func getThreat(description string, metadata map[string]interface{}) *pb.Threat {
 }
 
 func getSeverity(metadata map[string]interface{}) pb.Severity {
-	switch metadata["Severity"].(int) {
-	case 0:
-		return pb.Severity_INFO
-	case 1:
-		return pb.Severity_LOW
-	case 2:
-		return pb.Severity_MEDIUM
-	case 3:
-		return pb.Severity_HIGH
-	case 4:
-		return pb.Severity_CRITICAL
+	severityValue, ok := metadata["Severity"].(int)
+	if ok {
```

Comment:
nit:
```suggestion
	if !ok {
	    return -1
	}
```

Reply:
Thanks for the review. If we do so we still need do add another return in the final in case the switch is not satisfied (e.g: severityValue is 5).

Another thing, just realized it, but I think this func should follow the same logic as threat.pb.go:
https://github.com/aquasecurity/tracee/blob/cd6759e3c3e4abde2d50cf277fa59606461321b9/api/v1beta1/threat.pb.go#L136-L141

If any error is found, we should return Severity_INFO (0) and not (-1). If so, it should be:
```
func getSeverity(metadata map[string]interface{}) pb.Severity {
	severityValue, ok := metadata["Severity"].(int)
	if ok {
		switch severityValue {
		case 0:
			return pb.Severity_INFO
		case 1:
			return pb.Severity_LOW
		case 2:
			return pb.Severity_MEDIUM
		case 3:
			return pb.Severity_HIGH
		case 4:
			return pb.Severity_CRITICAL
		}
	}

	return pb.Severity_INFO
}
```

Reply:
Right. The nit here was just about removing the extra indentation of the switch case